cmd/govim/gopls_client.go

```diff
@@ -179,6 +179,7 @@ func (g *govimplugin) PublishDiagnostics(ctxt context.Context, params *protocol.
 	g.diagnosticsChanged = true
 	g.diagnosticsChangedQuickfix = true
 	g.diagnosticsChangedSigns = true
+	g.diagnosticsChangedHighlights = true
```

Comment:
At some point in the future I think it will make sense to shift this to by `uri` level granularity for whether diagnostics have changed, particularly when it comes to updating signs and highlights.

Reply:
Created https://github.com/govim/govim/issues/701 as a reminder

Reply:
Fully agree, thanks for creating the issue